addons/website/views/snippets/s_three_columns.xml

```diff
@@ -2,32 +2,38 @@
 <odoo>
 
 <template id="s_three_columns" name="Columns">
-    <section class="s_three_columns o_cc o_cc2 pt32 pb32 o_colored_level" data-vcss="001">
+    <section class="s_three_columns o_cc o_cc2 pt32 pb32 o_colored_level" data-vxml="001" data-vcss="001">
         <div class="container">
             <div class="row d-flex align-items-stretch">
-                <div class="col-lg-4 s_col_no_bgcolor pt16 pb16">
-                    <div class="card text-bg-white h-100">
-                        <img class="card-img-top" src="/web/image/website.s_three_columns_default_image_1" alt=""/>
+                <div data-name="Card" class="col-lg-4 s_col_no_bgcolor o_parent_handler pt16 pb16">
+                    <div class="s_card o_card_img_top card h-100 o_cc o_cc1 mt0" data-vxml="001">
+                        <figure class="o_card_img_wrapper ratio ratio-16x9 mb-0">
+                            <img class="o_card_img card-img-top" src="/web/image/website.s_three_columns_default_image_1" alt=""/>
+                        </figure>
                         <div class="card-body">
-                            <h3 class="card-title">Feature One</h3>
+                            <h5 class="card-title">Feature One</h5>
```

Comment:
🚨 _Design veto_ 🚨 🙅🏻‍♂️😅

By default we want cards to use `<h5>`.  Cards titles are not "section titles", you'll see designs consistent with our choice in other design-system as well (BS, tailwind, IBM Carbon,   etc..).

Reply:
Okay, good to know :slightly_smiling_face:

Reply:
@stefanorigano Are you speaking from a *design* point of view or not? I don't see how we can justify adding a h5: how would those 3 columns be below a h1 -> h2 -> h3 -> h4 ? IMO, only h2 should be used in snippets, and some h1. Of course h3 if there is a h2 before etc but that's it. A main snippet using h5 only does not make sense to me.

If it's from a design point of view, then `h2.h5-fs` while waiting for better, no? :thinking:

Reply:
> Are you speaking from a design point of view or not? I don't see how we can justify adding a h5 [...]

@qsm-odoo , I don't know how we can justify this structure to a screen reader, indeed... but I don't see neither how we can justify to the **user** that, by tweaking `<h2>`'s margins and line-height, this tiny title is be affected as well 🤷🏻‍♂️

<img width="665" alt="image" src="https://github.com/user-attachments/assets/f4c57f26-e2a4-401d-ab09-ea5d3fe319c7">


If the behaviour described  could be considered more or less "predictable" for section's flow content, it doesn’t makes a lot of sense for something that's **visually** presented as a self-contained element 😕

Having said that, and by a design point of view, we should allow a bit of flexibility considering that this is an editor. 
What if naif users set all the headings in a page to H4 for aesthetic reasons?
And, on the opposite side of the user-base spectrum , how "people that care"  could debug a page? Click on each little title to ensure that what looks like an h5 it's actually an h1? 😕  

I know that I'm not saying nothing new here but, while waiting for better, I have to design with what I have.
Using `<h5>` looks like the better trade possible considering how the system behaves today.

Reply:
> Ok, I guess nobody agrees with me on this anymore anyway, so I guess it's best to follow the majority's opinion. 

Sorry but there is a misunderstanding 🤔😅 I totally agree with you, we have to ensure accessibility and headings' nesting it's important, I'm not against that obviously 🙂
The point was just about what approach to use in order to avoid editor's side-effects.

---
   
> > Click on each little title to ensure that what looks like an h5 it's actually not an h1? 😕

>... well they exactly won't have to do that because we would not have used an h5 nor an h1 but an h2 🤔 

By bad, the scenario was not clear. Let's assume that we have a snippet using the following headings hierarchy:
```html
<h2 class="h4-fs">Level 1</h2>
<h3 class="h5-fs">Level 2</h3>
<h4 class="h6-fs">Level 3</h4>
<h3 class="h5-fs">Level 2</h3>
```
If the user sets the first `<h2>` to `<h1>` or removes the title entirely, then the entire structure it's invalidated. We're all aware of that of course, and providing a faulty straucture by default doesn’t help, I agree 😕

But, IMO,  the issue it's minor compared to the unpredictable visual glitch I posted before, because:
- people who don’t care will mess-up greatly regardless by the tags provided by default
- people who care will have to double-check manually as soon as the content is edited anyway. 



---

>  that's why I wanted *-fs classes to also impact line-height last year but...

I briefly discussed that with my team today and we have some doubts. It could solve that specific issue but we think that there may be other undesired effect. We're considering other approach that could make sense, at least by a design standpoint... but as said multiple times, we can discuss after the release 😅😉

Reply:
Yes, agreed, for now, let's just use what you feel is better when it comes to small titles. People next year will definitely discuss it :+1: